exercises/concept/lucians-luscious-lasagna/.meta/Exemplar.hs

```diff
@@ -0,0 +1,13 @@
+module LuciansLusciousLasagna exposing (elapsedTimeInMinutes, expectedMinutesInOven, preparationTimeInMinutes)
+
+
+expectedMinutesInOven =
```

Comment:
also under the idea that the exemplar should be something students should strive toward, I would thus say the exemplar should have types for these three functions

Reply:
I intend to introduce types in the next concept. That's why they are not included here.